### PR TITLE
feat: Show keyId after creating a new key

### LINKED ISSUE #2106: key_id should be displayed after creating new API key
### Preliminary Checks

- [X] I have reviewed https://unkey.com/docs for existing features that would solve my problem

- [X] I have searched for existing feature requests: https://github.com/unkeyed/unkey/issues

- [X] This issue is not a question, general help request, or anything other than a feature request directly related to Unkey. Please ask questions in our Discord community: https://unkey.com/discord.

### Is your feature request related to a problem? Please describe.

Currently after creating a new key, we are displaying the `CURL code` snippet and the `API key`.  But to access the key_id we need to click on the key details which is an annoying process. 

image

### Describe the solution

It would be better if we could have key_id displayed on the same page. 

### Describe alternatives you have considered (if any)

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by harshsbhat**
## What does this PR do?

Fixes #2106 

It displays the keyId after creating a new key. 
image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read [Contributing Guide](./CONTRIBUTING.md)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand areas
- [x] Ran `pnpm build`
- [x] Ran `pnpm fmt`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [x] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Unkey Docs if changes were necessary

## Summary by CodeRabbit

- **New Features**
	- Enhanced user interface for displaying API key information in the `CreateKey` component.
	- Key ID is now presented within a `Code` component for improved clarity.
	- Introduced a copy button for easy copying of the key ID.
	- Retained alert message emphasizing key visibility limitations.

### CODE REVIEW

**Review by chronark — CHANGES_REQUESTED** [review]
I think this placement may be too confusing. We already observe users mixing up `key` and `keyId` and displaying it with equal hierarchy will likely just amplify this.

A better place would be where the apiId is displayed right now, but that requires reworking the layout

**Comment by harshsbhat** [comment]
> I think this placement may be too confusing. We already observe users mixing up `key` and `keyId` and displaying it with equal hierarchy will likely just amplify this.
> 
> A better place would be where the apiId is displayed right now, but that requires reworking the layout

image

What do you think about this one?

**Comment by chronark** [comment]
that works 👍

**Comment by harshsbhat** [comment]
@chronark 
Yeah, I have also observed that many people are confused between Key and KeyID. Is it possible to change the prefix of keyId from `key_123 `to either `keyId_123` or `key_id_123`? I think it's too late to change this now. It will create many consistency issues with the keys that have already been issued. But wdyt?

**Comment by chronark** [comment]
I don't think that would cause consistency issues, what is your concern?

**Comment by harshsbhat** [comment]
PC:
image

Phone:
image

this is the new design 

I have pushed the changes

> that works 👍

File: apps/dashboard/app/(app)/apis/[apiId]/keys/[keyAuthId]/new/client.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import { revalidate } from "@/app/actions";
 import { CopyButton } from "@/components/dashboard/copy-button";
 import { Loading } from "@/components/dashboard/loading";
+import { PageHeader } from "@/components/dashboard/page-header";
```
Comment (chronark) [inline apps/dashboard/app/(app)/apis/[apiId]/keys/[keyAuthId]/new/client.tsx:5]: is this used?

**Review by chronark — CHANGES_REQUESTED** [review]

File: apps/dashboard/app/(app)/apis/[apiId]/keys/[keyAuthId]/new/client.tsx (hunk 1 above)
Comment (harshsbhat) [reply apps/dashboard/app/(app)/apis/[apiId]/keys/[keyAuthId]/new/client.tsx]: Yes, it is unused. I was planning to use it. But ended up using the `Code` component instead. Removed it

**Review by chronark — APPROVED** [review]

**Pull request merged**